aspnetcore/index.yml

```diff
@@ -89,15 +65,25 @@ conceptualContent:
         - url: web-api/action-return-types.md
           itemType: concept
           text: "Controller action return types"
-        - url: web-api/advanced/formatting.md
-          itemType: concept
-          text: "Format response data"
-        - url: web-api/handle-errors.md
+        - url: fundamentals/apis.md
+          itemType: overview
+          text: "See more"
+    # Card
+    - title: "Interactive client-side Blazor apps"
+      summary: Develop with reusable UI components that can take advantage of WebAssembly for near-native performance
+      links:
+        - url: blazor/index.md
+          itemType: overview
+          text: "Overview"
+        - url: https://dotnet.microsoft.com/learn/aspnet/blazor-tutorial/intro
+          itemType: get-started
+          text: "Build your first Blazor app"
+        - url: blazor/tutorials/build-a-blazor-app.md
+          itemType: get-started
+          text: "Build your first Blazor app with reusable components"
+        - url: blazor/hosting-models.md
           itemType: concept
-          text: "Handle errors"
-        - url: tutorials/web-api-javascript.md
-          itemType: tutorial
-          text: "Call an ASP.NET Core web API with JavaScript"
+          text: "Blazor hosting models"
```

Comment:
Seems like this would fit right after the overview.

Reply:
OK thanks.  Moving Blazor hosting models to right after the overview.